tunnel/tunnel_normal.go

```diff
@@ -272,3 +290,7 @@ func (t NormalTunnel) logger() *logrus.Entry {
 		"tunnel_id":   t.ID.String(),
 	})
 }
+
+func (t NormalTunnel) GetError() string {
```

Comment:
Is the convention that we're using getters?

Reply:
this DB column is nullable, so I assume we get the zero value of the string in here? should this be a *string and return `nil` instead?

Reply:
I'm in favor of a getter function returning a nilable *string